modules/API/API.cpp

```diff
@@ -489,12 +489,15 @@ void API::init() {
 
         std::string cmd_set_limit = cmd_base + "set_limit_amps";
 
-        this->mqtt.subscribe(cmd_set_limit, [&evse, &r_evse_manager_energy_sink = this->r_evse_manager_energy_sink,
+        const auto& evse_energy_sink = this->r_evse_manager_energy_sink.at(evse_id);
+
+        this->mqtt.subscribe(cmd_set_limit, [&evse_manager_check = this->evse_manager_check,
+                                             &evse_energy_sink = evse_energy_sink,
```

Comment:
- `&evse_energy_sink = evse_energy_sink` can be written as just `&evse_energy_sink`
- `evse_id` is not needed anymore for capture

Reply:
im still on it ;)